src/components/AssetTableBody.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'; // used implicitly by JSX
```

Comment:
It sort of feels like this could be left in AssetTable (which hardly does anything at the moment)

Reply:
You could say the same about AssetList for that matter. Since AssetTableHeader is broken out, it makes sense to be for AssetTableBody to be so.